### DISCUSSION TITLE
[Feature] Make user management table fit according to email length.

### DISCUSSION THREAD

**Discussion by Dental1347** [immich-app/immich#2576] [Feature Request]
### The feature

Hi,

As per the title and image below. My suggestion would be to extent the size of the table if the email is longer. Right now this is what is happening on the web client:
![image](https://github.com/immich-app/immich/assets/44779473/4dd402f0-0587-4c91-b474-861163798e6d)

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by uhthomas** [comment]
Tracked in #3141.
